Source/GlideHQ/tc-1.1+/fxt1.c

```diff
@@ -755,7 +755,7 @@ fxt1_quantize_MIXED1 (dword *cc,
     /* left microtile */
     if (maxColL == -1) {
 	/* all transparent black */
-	cc[0] = ~0UL;
+	cc[0] = 0xFFFFFFFFUL;
```

Comment:
Does this actually change the value that will be set to `cc[0]`, or is it only a style thing?